build.js

```diff
@@ -306,7 +330,11 @@ function moveBuildResult(builtPath, buildEnvironment) {
     process.exit(1);
   }
 
-  if (args.targetPlatform === "darwin") {
+  if (args.targetPlatform === "darwin" && args.targetArch === "x64") {
+    patchLibs(buildEnvironment.PATCH_TOOL, patchLibPath, builtPath);
+  }
+
+  if (args.targetPlatform === "darwin" && args.targetArch === "arm64") {
```

Comment:
The bodies of the `if` statements are identical:

```suggestion
  if (args.targetPlatform === "darwin") {
```